e2e-tests/vars.sh

```diff
@@ -11,7 +11,7 @@ export GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 export VERSION=${VERSION:-$(echo "${GIT_BRANCH}" | sed -e 's^/^-^g; s^[.]^-^g;' | tr '[:upper:]' '[:lower:]')}
 
 export IMAGE=${IMAGE:-"perconalab/percona-server-mysql-operator:${VERSION}"}
-export IMAGE_MYSQL=${IMAGE_MYSQL:-"perconalab/percona-server-mysql-operator:main-ps8.0"}
+export IMAGE_MYSQL=${IMAGE_MYSQL:-"percona/percona-server:8.0.25"}
```

Comment:
Why using a specific release image?

Reply:
this is the default version we use in cr.yaml. Rather than maintaining our own PS image, I'd like to use official images

Reply:
But wasn't the idea with using the main images to always use the latest version so that we don't need to change it manually in our branch?

Reply:
Yes, I get your point. Let's discuss this in today's call.